priam/src/main/java/com/netflix/priam/merics/CompactionMeasurement.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.priam.merics;
+
+/**
+ * Measurement class for scheduled compactions
+ * Created by aagrawal on 2/28/18.
+ */
+
+public class CompactionMeasurement implements IMeasurement<Object> {
```

Comment:
We should start thinking about using spectator instead of our own measurements.

Reply:
I will open a separate JIRA for that as it will touch more components.

Reply:
👍  We can go iterative on that, as we are touching any classes we can fix it.